### PR TITLE
Developer Workflow Documentation: Code Review

### PR DESCRIPTION
**Pull request by mts299**
# Scope

This documentation focuses on Code Reviews in GitHub.  

Please read and make sure it makes sense.

### CODE REVIEW

**Review by ecbland — COMMENTED** [review]
This looks good @mts299 :100: 
I hope I haven't failed too much with the "don't be too pedantic" and "avoid personal preference" stuff...feel free to disagree on anything :smile:

File: docs/dev/code_reviews.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+If you don't agree with a suggestion try to find a compromise or acknowledge their effort and explain why you prefer your style.
+Remember to hit `Start a review` to prevent multiple email notifications. 
+
+If you made the change requested, then select `resolve conflict` so they know you addressed their comment.
+
+If you are answering a question or responding to some feedback, remember to acknowledge their time and effort, then address it in collaborative manner. 
+Remember to hit `Start a review` to prevent multiple email notifications. 
+
+Once all comments are addressed then follow steps 9-10 in the `How to start a Code Review` section.
```
[the remark this answers is not in the document]
Comment (mts299) [reply docs/dev/code_reviews.md:115]: will rebuild it again and double check

**Comment by mts299** [comment]
@ecbland updated the code to fix the numbering issue

**Comment by mts299** [comment]
Should be good to merge if you think it is good.

**Pull request merged**